tcms/rpc/tests/utils.py

```diff
@@ -20,7 +20,7 @@ class XmlrpcAPIBaseTest(test.LiveServerTestCase):
     def _fixture_setup(self):
         # restore the serialized data from initial migrations
         # this includes default groups and permissions
-        super(XmlrpcAPIBaseTest, self)._fixture_setup()
+        super(APITestCase, self)._fixture_setup()
```

Comment:
Use the Python 3 form of super which is `super()._fixture_setup()`. In Python 3 you don't have to pass the current class name & self unless you want to skip to another part of the inheritance chain or maybe there is multiple inheritance involved.